packages/vm/tests/GeneralStateTestsRunner.ts

```diff
@@ -34,6 +34,13 @@ function parseTestCases(
       tx.gasLimit = testData.transaction.gasLimit[testIndexes['gas']]
       tx.value = testData.transaction.value[testIndexes['value']]
 
+      if (tx.accessLists) {
+        tx.accessList = testData.transaction.accessLists[testIndexes['data']]
+        if (tx.chainId == undefined) {
+          tx.chainId = 1
```

Comment:
This might look weird, but these state tests don't have chain ID in their tests, so have added manually (also raised a point about this on the tests PR)